assets/js/common/ActivityLogsSettingsModal/ActivityLogsSettingsModal.stories.jsx

```diff
@@ -0,0 +1,45 @@
+import ActivityLogsSettingsModal from '.';
+
+export default {
+  title: 'Components/ActivityLogsSettingsModal',
+  component: ActivityLogsSettingsModal,
+  argTypes: {
+    open: {
+      description: 'Whether the dialog is open or not',
+      control: {
+        type: 'boolean',
+      },
+    },
+    initialRetentionTime: {
+      description:
+        'A structured `{value,unit}` object that defines a retention time interval. `unit` is one of `days`, `months`, `years`.',
+      control: {
+        type: 'object',
+      },
+    },
+    errors: {
+      description: 'The validation error to be shown',
+    },
+  },
+};
+
+export const Default = {
+  args: {
+    open: false,
+    initialRetentionTime: { value: 1, unit: 'months' },
+    errors: [],
+  },
+};
+
+export const WithErrors = {
```

Comment:
This story cannot render because there is no `initialRetentionTime` set.

I am getting a `TypeError: Cannot read properties of undefined (reading 'value')`

Besides this, after adding a `initialRetentionTime: {}` to make it render, there might be some css to adjust wrt spacing between the input field and the error message.

![image](https://github.com/trento-project/web/assets/8167114/22c5a3a1-17a8-4d03-884e-e0b8135be6b2)

Reply:
Fixed in https://github.com/trento-project/web/pull/2706/commits/394c0121d1b9a76269a7dfcfd7cc112ea8e9fd9b